modular_zubbers/code/modules/loadout/categories/pocket.dm

```diff
@@ -70,6 +70,15 @@
 	name = "Pocket Watch"
 	item_path = /obj/item/clothing/accessory/pocketwatch
 
+/datum/loadout_item/pocket_items/mimezippo
+	name = "Pale Zippo"
+	item_path = /obj/item/lighter/royal
+	restricted_roles = list(JOB_MIME)
```

Comment:
Already available as the "Royal Zippo" lighter

```suggestion
```

Reply:
WOOPS wrong filepath